test/mosaic-integration-tests/src/test/java/org/eclipse/mosaic/test/junit/MosaicSimulationRule.java

```diff
@@ -196,9 +196,12 @@ public MosaicSimulation.SimulationResult executeSimulation(Path scenarioDirector
 
     public MosaicSimulation.SimulationResult executeSimulation(Path scenarioDirectory, String config) {
         try {
+            File f = scenarioDirectory.resolve(config).toFile();
+            if (!f.exists() || !f.isFile()) {
+                throw new InstantiationException("Scenario config file not found.");
+            }
             return executeSimulation(scenarioDirectory,
-                    new ObjectInstantiation<>(CScenario.class)
-                            .readFile(scenarioDirectory.resolve(config).toFile())
+                    new ObjectInstantiation<>(CScenario.class).readFile(f)
```

Comment:
This formatting doesn't make sense to me.

IMO it should either be
```suggestion
            return executeSimulation(scenarioDirectory,
                    new ObjectInstantiation<>(CScenario.class).readFile(f));
```
or (personal preference)
```suggestion
            return executeSimulation(scenarioDirectory, new ObjectInstantiation<>(CScenario.class).readFile(f));
```
or
```suggestion
            return executeSimulation(
                    scenarioDirectory, 
                    new ObjectInstantiation<>(CScenario.class).readFile(f)
            );
```